src/math/Kconfig

```diff
@@ -38,6 +38,15 @@ config SQRT_FIXED
 	  to  calculate square root.square function having positive number
 	  y as input and return the positive number x multiplied by itself (squared)
 
+config EXP_FIXED
+	bool "Exponential  functions"
+	default n
+	help
+	  By selecting this, the 32-bit exp fcn() function can be used to calculate
```

Comment:
Better to change "exp fcn()" -> "exp_int32()" to know what the function name is that this kconfig option provides.